deviceModeInit.js

```diff
@@ -153,32 +159,40 @@ var rudderTracking = (function () {
 
   function checkCartAttributes(cart) {
     const { attributes } = cart;
-    if (attributes?.rudderAnonymousId && attributes?.rudderUpdatedAt) {
-      return true;
+    if (attributes?.rudderAnonymousId) {
+      return false;
     }
-    return false;
+    return true;
   }
 
+  function updateCookies() {
+    const cookieOptions = {
+      action: "set",
+      expire_hr: 2,
+      name: "rs_shopify_updated_at",
+      value: `${Date.now()}`
+    }
+    cookie_action(cookieOptions);
+  }
   function updateCartAttribute() {
     const anonymousId = rudderanalytics.getAnonymousId();
     return rs$.post(
       window.Shopify.routes.root + "cart/update.json",
       {
         attributes: {
           rudderAnonymousId: anonymousId,
-          rudderUpdatedAt: Date.now(),
         },
       },
       undefined,
       "json"
     );
   }
-
-  function checkUpdateTime(cart) {
+  function checkUpdateTime() {
     const oneHourTimeInMilliSeconds = 60 * 60 * 1000;
-    const { rudderUpdatedAt } = cart.attributes;
     const currentTime = Date.now();
-    return currentTime - rudderUpdatedAt > oneHourTimeInMilliSeconds;
+    const cookies = cookie_parse();
+    const lastupdatedValue = cookies?.rs_shopify_updated_at || 0;
```

Comment:
Use `cookie_action` with `get` parameter to read cookies check line 142

Reply:
Done